scss/_base_forms.scss

```diff
@@ -1,5 +1,6 @@
 @import 'settings';
 @import 'base_forms-tick-elements';
+@import 'patterns_icons';
```

Comment:
This feels a bit weird. Base styles depend on a "pattern". 
Maybe we should extract icon placeholders and make them part of "base" styles.

Reply:
I agree or see what icon it requires. Could that to moved local to the base_forms?

Reply:
@anthonydillon Base forms use chevron icon.

There are also other patterns that include "patterns_icons" (accortion for plus/minus, code copyable for copy icon). So it seems like at least the icon mixins (not `p-icon` pattern itself) should be part of base/global Vanilla, to be accessible from any pattern independently.